packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -182,6 +182,42 @@ typedef DecoderCallback = Future<ui.Codec> Function(Uint8List bytes, {int cacheW
 ///
 /// The following image formats are supported: {@macro flutter.dart:ui.imageFormats}
 ///
+/// ## Image resolving
```

Comment:
nit: Maybe just make this heading "Lifecycle"? Or "Lifecycle of resolving an image" or some such?